EndlessClient/Dialogs/Factories/ItemTransferDialogFactory.cs

```diff
@@ -44,13 +44,21 @@ public ItemTransferDialog CreateItemTransferDialog(string itemName, ItemTransfer
                 transferType,
                 totalAmount,
                 message);
-            dlg.DialogClosing += (_, _) => _sfxPlayer.PlaySfx(SoundEffectID.DialogButtonClick);
+
+            if (playDefaultSound)
```

Comment:
It looks like the official client doesn't play any sound for the "ok" button. I checked "drop", "junk", "buy", and "sell". I'm good with either:

1) Only play the `DialogButtonClick` sound on "cancel" button press
2) Play `DialogButtonClick` on either button. Play `DialogButtonClick` and safety comment message together

I don't really care too much about the sounds overlapping in this case. I'd rather avoid having a branching arg in the method signature for which sound is played.

That means the options for this file are:
1) Undo all the changes, then check which button was pressed and only play the default sfx if the button is "cancel"
2) Undo all the changes and leave this file untouched

Reply:
I'd rather go for 1) as that's what I initially made me include playDefaultSound: 

It would mean the inclusion of a new using statement along with 
![image](https://github.com/ethanmoffat/EndlessClient/assets/70440468/d6bebe92-3e19-4028-bb2b-d1d4c4b1c35b)

Would this be okay?

Best,
Stephen